userapi/src/main/java/fk/prof/userapi/UserapiConfigManager.java

```diff
@@ -4,55 +4,39 @@
 import com.google.common.io.Files;
 import io.vertx.core.json.JsonObject;
 
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
 import java.io.File;
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
 import java.util.Properties;
+import java.util.Set;
+import java.util.stream.Collectors;
 
 /**
- * Map configuration to POJO and encapsulate default settings inside it
- * Use this POJO to access config and remove jsonobject getters from rest of the code base
+ * Map configuration to {@link Configuration} and encapsulate default settings inside it.
  */
 public class UserapiConfigManager {
```

Comment:
I am actually inclined to merge this class with Configuration class. This class only has helper methods. 
I am also reluctant to move small common code to a common module as of now as there is there is no such module for this purpose.
What do you say.